### ISSUE TITLE
Support setting the tooltip text as an attribute

### ISSUE THREAD

**Issue by ergo** [PolymerElements/paper-tooltip#84]
Hello,

Would it be possible to use paper tooltip similar way to the paper-toast?
I would love to be able to use it like:

    <paper-tooltip text="I'm a tooltip!">

Or even read the title property of the element the tooltip attaches to programmaticly.

**Comment by notwaldorf** [comment]
You can use bindings without having the tooltip as an attribute: http://jsbin.com/woqoleh/edit?html,output

Does that work for you?

**Comment by notwaldorf** [comment]
(I do agree that as a feature request using the target's title would be nice. I'll put something together!)

**Comment by ergo** [comment]
@notwaldorf I looked at your jsbin, thing is i want to avoid light dom (FUOC) showing up before elements are loaded if possible. I'm trying to use polymer on a non SPA application so I cant use `unresolved` on body globally.

**Comment by ergo** [comment]
@notwaldorf I could try to submit a PR for new functionality, but i think that best option to implement that for pages that have 100 tooltips or more would be to have something like `is="tooltip-provider"`. I'm not even sure if that kind of approach is possible.

**Comment by notwaldorf** [comment]
@ergo have you tried adding `<body unresolved>`, to stop the FOUC?

**Comment by ergo** [comment]
@notwaldorf Yes, but I'm adding elements to a non-SPA application, so hiding whole body and showing it via `unresolved` end with one big flash of rendered html.

If I could use `unresolved` on individual elements then it would solve my issue (same as ng-cloak directive works in angular). But seems this strategy is not supported by polymer 1.6 as of now.

**Comment by ergo** [comment]
@notwaldorf I was also thinking that maybe for web application another option (or another element) that would behave like `paper-toast` would make sense - that allows you to place a single top level tooltip element, and using events just show it on hover when required. This would also solve the problem of compoment encapsulation right? So event could do something like:

   tooltipElem.text = 'foo'
   tooltipElem.for = event.target

so the tooltip would work and position on both light and shadow dom?
